packages/@react-stately/color/src/Color.ts

```diff
@@ -61,6 +61,8 @@ abstract class Color implements IColor {
   getChannelName(channel: ColorChannel, locale: string) {
```

Comment:
When importing `clamp` should be use the @react-stately package?
```ts
import {clamp} from '@react-stately/utils';
```
instead of
```ts
import {clamp} from '@react-aria/utils';
```?